classes/components/listPanels/PKPContributorsListPanel.inc.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @file classes/components/listPanels/PKPAnnouncementsListPanel.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPAnnouncementsListPanel
+ * @ingroup classes_components_list
+ *
+ * @brief A ListPanel component for viewing and editing email templates
+ */
+
+namespace PKP\components\listPanels;
+
+class PKPContributorsListPanel extends ListPanel
+{
+    /** @var string URL to the API endpoint where items can be retrieved */
+    public $apiUrl = '';
+
+    /** @var int How many items to display on one page in this list */
+    public $count = 30;
+
+    /** @param \PKP\components\forms\announcement\PKPAnnouncementForm Form for adding or editing an email template */
+    public $form = null;
+
+    /** @var array Query parameters to pass if this list executes GET requests  */
+    public $getParams = [];
+
+    /** @var int Max number of items available to display in this list panel  */
+    public $itemsMax = [];
+
+    /** @var string URL to the publication API endpoint */
+    public $publicationApiUrl = '';
+
+    /**
+     * Initialize the form with config parameters
+     *
+     * @param $id string
+     * @param $title string
+     * @param $args array Configuration params
+     */
+    public function __construct($id, $title, $args = [])
+    {
+        parent::__construct($id, $title, $args);
+    }
+
+    /**
+     * @copydoc ListPanel::getConfig()
+     */
+    public function getConfig()
+    {
+        \AppLocale::requireComponents(LOCALE_COMPONENT_PKP_MANAGER);
+        \AppLocale::requireComponents(LOCALE_COMPONENT_APP_MANAGER);
+        $request = \Application::get()->getRequest();
+
+        $config = parent::getConfig();
+
+        $config = array_merge(
+            $config,
+            [
+                'addContributorLabel' => __('grid.action.addContributor'),
+                'apiUrl' => $this->apiUrl,
+                'publicationApiUrl' => $this->publicationApiUrl,
```

Comment:
Do we need `apiUrl` anymore if we have `publicationApiUrl`? I think every URL used by `ContributorsListPanel` can be constructed from `publicationApiUrl`